man/ctags-incompatibilities.7.rst.in

```diff
@@ -86,14 +89,80 @@ removed in the future.
 The former form may be friendly to shell completion engines.
 
 Disallowing to define a kind with "file" as name
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+....................................................................................
 
 The kind name "file" is reserved.  Using it as part of kind spec in
 ``--regex-<LANG>`` option is now disallowed.
 
+Disallowing to define a kind with "F" as letter
+....................................................................................
+
+The kind letter "F" is reserved.  Using it as part of a kind spec in
+``--regex-<LANG>`` option is now disallowed.
+
+Disallowing to use other than alphabetical character as kind letter
+....................................................................................
+
+Exuberant-ctags accepts a character other than alphabetical character
+as kind letter in ``--regex-<LANG>=...`` option.  Universal-ctags
+accepts only an alphabetical character.
+
+Acceptable characters as parts of a kind name
+....................................................................................
+
+Exuberant-ctags accepts any character as a part of a kind name
+defined with ``--regex-<LANG>=/regex/replacement/kind-spec/``.
+
+Universal-ctags accepts only an alphabetical character as
+the initial letter of a kind name.
+Universal-ctags accepts only an alphabetical character or
+numerical character as the rest letters.
+
+An example::
+
+  --regex-Foo=/abstract +class +([a-z]+)/\1/a,abstract class/i
+
+Universal-ctags rejects this because the kind name, "abstract class",
+includes a whitespace character.
+
+This requirement is for making the output of Universal-ctags follow
+the tags file format.
+
+A combination of a kind letter and a kind name
+....................................................................................
+
+In Universal-ctags, the combination of
+a kind letter and a kind name must be unique in a language.
+
+You cannot define more than one kinds reusing a kind letter with
```

Comment:
minor: should be "more than one kind reusing" because "one kind" is singular (even though "more than" seems plural ...)

Reply:
(Reflected the suggestion to the local source tree).